### ISSUE TITLE
avis Home page! Menu Items are not in Focus Order

### ISSUE THREAD

**Issue by Chetna1307** [Kishan1601/Test-1#4]
Environment
UAT 2022-03
Windows 10
Chrome Version 98.0.4758.102 (Official Build) (64-bit)
NVDA 2021.3.5

Repro Steps
Press Tab Key to navigate to the  list option your focus will directly land on the 6th option which is Car Sales, using down arrow key to navigate to other list option but it will only go on the 7th list option and then out of the list  

Expected Result 
User should be able to navigate through all list items using the keyboard 

Actual Result 
Collapsible menu options list which have options like Reservation, Offers Locations etc. are not in proper focus order, 

WCAG Guideline
https://www.w3.org/TR/WCAG21/#focus-order
